src/main/java/spoon/support/reflect/declaration/CtAnonymousExecutableImpl.java

```diff
@@ -112,4 +103,63 @@ public boolean hasModifier(ModifierKind modifier) {
 		return (T) this;
 	}
 
+	@Override
+	public List<CtParameter<?>> getParameters() {
+		throw new UnsupportedOperationException("You can't have parameters in a anonymous executable");
+	}
+
+	@Override
+	public CtExecutable setParameters(List list) {
+		throw new UnsupportedOperationException("You can't have parameters in a anonymous executable");
+	}
+
+	@Override
+	public CtExecutable addParameter(CtParameter parameter) {
+		throw new UnsupportedOperationException("You can't have parameters in a anonymous executable");
+	}
+
+	@Override
+	public boolean removeParameter(CtParameter parameter) {
+		throw new UnsupportedOperationException("You can't have parameters in a anonymous executable");
+	}
+
+	@Override
+	public Set<CtTypeReference<? extends Throwable>> getThrownTypes() {
+		throw new UnsupportedOperationException("You can't have throw types in a anonymous executable");
+	}
+
+	@Override
+	public CtExecutable setThrownTypes(Set thrownTypes) {
+		throw new UnsupportedOperationException("You can't have throw types in a anonymous executable");
+	}
+
+	@Override
+	public CtExecutable addThrownType(CtTypeReference throwType) {
+		throw new UnsupportedOperationException("You can't have throw types in a anonymous executable");
+	}
+
+	@Override
+	public boolean removeThrownType(CtTypeReference throwType) {
+		throw new UnsupportedOperationException("You can't have throw types in a anonymous executable");
+	}
+
+	@Override
+	public String getSimpleName() {
+		throw new UnsupportedOperationException("You can't have a name in a anonymous executable");
```

Comment:
breaks clieant code, should return empty string or "<static>" (inspired from "<init>")